lib/src/airtable_base.dart

```diff
@@ -50,28 +50,31 @@ class Airtable {
   ///
   /// [Returns AirtableRecord with ids when successful]
   /// [Returns null when unsuccessful]
-  Future<AirtableRecord> createRecord(
-    String recordName,
-    AirtableRecord record,
-  ) async {
-    final records = await createRecords(recordName, [record]);
+  Future<AirtableRecord> createRecord(String recordName, AirtableRecord record,
+      {bool typecast = false}) async {
+    final records =
+        await createRecords(recordName, [record], typecast: typecast);
     return records == null || records.isEmpty ? null : records.first;
   }
 
   /// Creates multiple records in Airtable using a list of AirtableRecord instances
   ///
   Future<List<AirtableRecord>> createRecords(
-    String recordName,
-    List<AirtableRecord> records,
-  ) async {
-    final body = {'records': records.map((record) => record.toJSON()).toList()};
+      String recordName, List<AirtableRecord> records,
+      {bool typecast = false}) async {
+    final body = {
+      'records': records.map((record) => record.toJSON()).toList(),
+      'typecast': typecast,
+    };
 
     final response = await client.post(
       _recordApiUrl(recordName),
       headers: _headers,
       body: jsonEncode(body),
     );
 
+    print(response.body);
```

Comment:
I might remove this in a follow-up unless you find this helpful for some reason. If it's helpful, I might add a flag for logging.

Reply:
@deriegle it was an oversight from some debugging. You can remove the print.

Also thanks for being so fast with the merge this time.